tools/universe/version_resolver.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+"""
+This module can be used to determine the latest version of a particular package
+in the Universe.
+"""
+
+
+class Version:
+    """Encapsulates the releases-package version pair."""
+    def __init__(self, release_version, package_version):
+        self.release_version = release_version
+        self.package_version = package_version
+
+    def __eq__(self, other):
+        if self.release_version != other.release_version:
+            return False
+
+        return self.package_version == other.package_version
+
+    def __lt__(self, other):
+        if self.release_version < other.release_version:
+            return True
+
+        return self.package_version < other.package_version
+
+    def __gt__(self, other):
+        if self.release_version > other.release_version:
+            return True
+
+        return self.package_version > other.package_version
+
+    def __str__(self):
+        return str(self.package_version)
+
+
+class VersionResolver:
```

Comment:
This feels pretty tied to the `PackageManager` implementation, in terms of expecting the Universe response data to match a certain format. It may make things a little simpler to just combine the two into a single class that does any version extraction internally